config/resources/change-events.json

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
I moved it into a separate file, but I'm not sure if this is the best way to organize it since "decisions" aren't only related to the change events.

Reply:
The domain.json is getting a bit too big, so I think it's good to split it up.
I guess we'll find the right granularity through trial and error. So far I think it's sensible to have the decisions in here. Splitting things up per class could be a bit fragmented.